src/cmake-tools.ts

```diff
@@ -793,19 +829,9 @@ export class CMakeTools implements vscode.Disposable, api.CMakeToolsAPI {
           });
       return null;
     }
-    // Ensure that we've configured the project already. If we haven't, `getOrSelectLaunchTarget` won't see any
-    // executable targets and may show an uneccessary prompt to the user
-    if (await this._needsReconfigure()) {
-      const rc = await this.configure();
-      if (rc !== 0) {
-        log.debug('Configuration of project failed.');
-        return null;
-      }
-    }
-    const target = await this.getOrSelectLaunchTarget();
-    if (!target) {
-      // The user has nothing selected and cancelled the prompt to select a target.
-      log.debug('No target selected.');
+
+    const targetExecutable = await this.PrepareLaunchTargetExecutable();
```

Comment:
Compile error because this var was renamed.

Reply:
Done